webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -178,16 +189,29 @@ export default class TicketPopover extends React.PureComponent<Props, State> {
         }
 
         const {ticketDetails, href: jiraTicketURI} = this.state;
-        const jiraAvatar = ticketDetails && ticketDetails.jiraIcon;
-        const jiraIssueIconURI = ticketDetails && ticketDetails.issueIcon;
-        const jiraTicketKey = ticketDetails && ticketDetails.ticketId;
-        const jiraTicketTitle = ticketDetails && ticketDetails.summary;
-        const jiraTicketAssigneeAvatarURI = ticketDetails && ticketDetails.assigneeAvatar;
-        const jiraTicketAssigneeName = ticketDetails && ticketDetails.assigneeName;
-        const jiraTicketStatusName = ticketDetails && ticketDetails.statusKey;
-        const jiraTicketDescription = ticketDetails && ticketDetails.description;
-        const jiraTicketVersions = ticketDetails && ticketDetails.versions;
-        const jiraTicketLabels = ticketDetails && ticketDetails.labels;
+
+        let jiraAvatar = null;
+        let jiraIssueIconURI = null;
+        let jiraTicketKey = null;
+        let jiraTicketTitle = null;
+        let jiraTicketAssigneeAvatarURI = null;
+        let jiraTicketAssigneeName = null;
+        let jiraTicketStatusName = null;
+        let jiraTicketDescription = null;
+        let jiraTicketVersions = null;
+        let jiraTicketLabels = null;
+        if (ticketDetails) {
+            jiraAvatar = ticketDetails.jiraIcon;
+            jiraIssueIconURI = ticketDetails.issueIcon;
+            jiraTicketKey = ticketDetails.ticketId;
+            jiraTicketTitle = ticketDetails.summary;
+            jiraTicketAssigneeAvatarURI = ticketDetails.assigneeAvatar;
+            jiraTicketAssigneeName = ticketDetails.assigneeName;
+            jiraTicketStatusName = ticketDetails.statusKey;
+            jiraTicketDescription = ticketDetails.description;
+            jiraTicketVersions = ticketDetails.versions;
+            jiraTicketLabels = ticketDetails.labels;
+        }
```

Comment:
Do you see any specific reason for making them `null`? If not, I think we can just declare them in one line and they will have their value as `undefined`